threshold: 0.5

subscribers:
  
  queue_size: 100
  exact_time: false
  slop: 0.5

  image_rgb: usb_cam/image_raw

  #points: kinect2/qhd/points

publishers:

  face_recognition:
    topic: /butia_vision/br/face_recognition
    queue_size: 1

servers:

  list_classes:
    service: /butia_vision/br/people_detection/list_classes

  start:
    service: /butia_vision/br/people_detection/start

  stop:
    service: /butia_vision/br/people_detection/stop

names:
    starting: features
